src/CreateTwoDClustersFromThreeDAlgorithm.cc

```diff
@@ -48,6 +59,36 @@ StatusCode CreateTwoDClustersFromThreeDAlgorithm::Run()
     std::vector<PandoraContentApi::Cluster::Parameters> clustersV;
     std::vector<PandoraContentApi::Cluster::Parameters> clustersW;
 
+    // Populate a spatial map of CaloHit to 2D position.
+    SpatialHitMap spatialHitMapU, spatialHitMapV, spatialHitMapW;
+
+    for (const CaloHit *const pCaloHit : *pCaloHitListU)
+    {
+        if (!PandoraContentApi::IsAvailable(*this, pCaloHit))
+            continue;
+
+        const CartesianVector pos = pCaloHit->GetPositionVector();
+        spatialHitMapU[QuanitizePosition(pos.GetX(), pos.GetZ())] = pCaloHit;
```

Comment:
This needs to use the updated `QuantizePosition` name, here & other places where it is called.

Reply:
Apologies, the change got lost in the other "changed" files in my repo! Pushed up now.

(For LArContent etc I have a custom `.gitignore` to ignore my build folder, output files, editor cache files etc, which I don't here so I have ~100 "changed" files, so much easier to miss things!)